src/UserSpaceInstrumentation/TestUtils.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "TestUtils.h"
+
+#include <absl/strings/str_cat.h>
+#include <capstone/capstone.h>
+
+#include <string>
+
+#include "ObjectUtils/Address.h"
+#include "ObjectUtils/ElfFile.h"
+#include "ObjectUtils/LinuxMap.h"
+#include "OrbitBase/ExecutablePath.h"
+#include "OrbitBase/Logging.h"
+#include "OrbitBase/UniqueResource.h"
+
+namespace orbit_user_space_instrumentation {
+
+AddressRange GetFunctionAbsoluteAddressRangeOrDie(std::string_view function_name) {
+  auto modules = orbit_object_utils::ReadModules(getpid());
+  CHECK(!modules.has_error());
+  std::string module_file_path;
+  AddressRange address_range_code(0, 0);
+  for (const auto& module : modules.value()) {
+    if (module.file_path() == orbit_base::GetExecutablePath()) {
+      module_file_path = module.file_path();
+      address_range_code.start = module.address_start();
+      address_range_code.end = module.address_end();
+      break;
+    }
+  }
+  CHECK(!module_file_path.empty());
+  auto elf_file = orbit_object_utils::CreateElfFile(module_file_path);
+  CHECK(!elf_file.has_error());
+  auto syms = elf_file.value()->LoadDebugSymbols();
+  CHECK(!syms.has_error());
+  for (const auto& sym : syms.value().symbol_infos()) {
+    if (sym.name() == function_name) {
+      const uint64_t address = orbit_object_utils::SymbolVirtualAddressToAbsoluteAddress(
+          sym.address(), address_range_code.start, syms.value().load_bias(),
+          elf_file.value()->GetExecutableSegmentOffset());
+      const uint64_t size = sym.size();
+      return {address, address + size};
+    }
+  }
+  UNREACHABLE();
+  return AddressRange();
```

Comment:
I think that the compiler understands that you don't need this `return` after `UNREACHABLE();`.

Reply:
done